packages/loader/container-definitions/src/loader.ts

```diff
@@ -172,6 +172,13 @@ export interface IContainer extends IEventProvider<IContainerEvents>, IFluidRout
     request(request: IRequest): Promise<IResponse>;
 }
 
+/**
+ * Events emitted by the Loader "upwards" to the Host
+ */
+export interface ILoaderEvents extends IEvent {
```

Comment:
literally just filed an issue for this: #5114

Reply:
you commented the same in the issue i'm resolving against so i made sure to add it 😁